bom/runtime/pom.xml

```diff
@@ -195,6 +195,7 @@
         <scram-client.version>2.1</scram-client.version>
         <grpc-jprotoc.version>1.0.1</grpc-jprotoc.version>
         <grpc.version>1.29.0</grpc.version>
+        <picocli.version>4.2.0</picocli.version>
```

Comment:
The latest picocli version is now [4.3.2](https://github.com/remkop/picocli/releases/tag/v4.3.2).